nodeup/pkg/model/directories.go

```diff
@@ -42,8 +43,7 @@ func (b *DirectoryBuilder) Build(c *fi.ModelBuilderContext) error {
 				{"/bin/mount", "--bind", "/home/kubernetes/bin", "/home/kubernetes/bin"},
```

Comment:
Whilst you're in this file, could maybe substitute the paths here for `dir` variable defined above.

Reply:
good shout!